go-voice_registration.js

```diff
@@ -1394,8 +1394,31 @@ go.app = function() {
                         action: 'enter'
                     },[self.metric_prefix, "registrations_completed"].join('.')
                 )
+                .add.time_between_states(
+                    {
+                        state: 'state_personnel_auth',
+                        action: 'enter'
+                    },{
+                        state: 'state_end_voice',
+                        action: 'enter'
+                    }, [self.metric_prefix, 'time_to_register.avg'].join('.')
```

Comment:
We don't need the `.avg` here, because that is automatically added with the `im.metrics.fire.avg` below, so this would result in a metric name with `.avg.avg`